content/integrations/entrust-keycontrol.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Entrust KeyControl
+date: 2023-08-01
+lastmod: :git
+draft: false
+tableOfContents: true
+---
+
+[Entrust KeyControl](https://www.entrust.com/digital-security/key-management/keycontrol) is a proprietary KMS that provides a secret store that can be used by KES.
+
+```goat
+                +-----------------------------------------+
+ .----------.   |   .----------.    .------------------.  |
+| KES Client +--+--+ KES Server +--+ Entrust KeyControl | |
+ '----------'   |   '----------'    '------------------'  |
+                +-----------------------------------------+
+```
+
+## Prerequisites
+
+
+This guide assumes that you have setup an Entrust KeyControl v10.1 cluster. 
```

Comment:
Suggested alternative wording.

```suggestion
This procedure was written and tested against Entrust KeyControl v.10.1. 
The provided instructions may work against other KeyControl versions.
This procedure assumes you have prior experience with Entrust products and have followed their documentation, best practices, and other published materials in deploying the KeyControl service.
```

Reply:
Yeah, should have ripped that verbiage from elsewhere. This is good.